src/contracts/core/DelegationManager.sol

```diff
@@ -620,11 +620,6 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
             "DelegationManager.completeQueuedAction: action is not in queue"
         );
 
-        require(
-            withdrawal.startBlock + withdrawalDelayBlocks <= block.number, 
-            "DelegationManager.completeQueuedAction: withdrawalDelayBlocks period has not yet passed"
-        );
-
```

Comment:
This was enforcing a withdrawal delay on EigenPod shares and is no longer doing that -- that's not an intended change, unless I'm misunderstanding!

Reply:
I thought we still wanted a global "minimum withdrawal delay" as well, which it still seems appropriate to check here.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/392/commits/235957282bc952638c45078ae1ba28638f78263b#diff-d323417ded59a593f4ea410225a9167b24198c4d2c05cbc0fe9af47d34b88810R641
Added back and named it `minWithdrawalDelayBlocks`. Min number of blocks to pass to withdraw a strategy will be 
max(`minWithdrawalDelayBlocks`, `strategyWithdrawalDelayBlocks[strategy]`)